Documentation/Events/EventDispatcher/Index.rst

```diff
@@ -210,18 +210,32 @@ If an extension author wants to provide a custom Event Listener, an according en
      Vendor\MyExtension\EventListener\NullMailer:
        tags:
          - name: event.listener
+           method: handleEvent
            identifier: 'myListener'
            before: 'redirects, anotherIdentifier'
            event: TYPO3\CMS\Core\Mail\Event\AfterMailerInitializationEvent
 
 
 The tag name :yaml:`event.listener` identifies that a listener should be registered.
 
-The custom PHP class :php:`MyCompany\MyPackage\EventListener\NullMailer` serves as the listener,
-whereas the :yaml:`identifier` is a common name so orderings can be built upon the identifier,
-the optional :yaml:`before` and :yaml:`after` attributes allow for custom sorting against :yaml:`identifier`.
+The custom PHP class :php:`MyCompany\MyPackage\EventListener\NullMailer` serves as the listener
+whose :php:`handleEvent` method is called once the :yaml:`event` is dispatched.
+The :yaml:`identifier` is a common name so orderings can be built upon the identifier,
+the optional :yaml:`before` and :yaml:`after` attributes allow for custom sorting against the
+:yaml:`identifier` of other listeners.
 
-If no attribute :yaml:`method` is given, the class is treated as Invokable, thus :php:`__invoke` method is called.
+If no attribute :yaml:`method` is given, the class is treated as invokable, thus its :php:`__invoke` method will be called:
+
+.. code-block:: yaml
+   :caption: EXT:some_extension/Configuration/Services.yaml
```

Comment:
```suggestion
   :caption: EXT:my_extension/Configuration/Services.yaml
```

Reply:
This is a copy from the snippet above, so we should adjust both.